.github/workflows/bazel_build_bindings.yml

```diff
@@ -23,42 +23,17 @@ on:
 jobs:
   linux:
     runs-on: ubuntu-18.04
-    env:
-      CXX: clang++
-      CC: clang
-      PYTHON_BIN: /usr/bin/python3
-      # Install the version of Bazel that is required by our .bazelversion file.
-      BAZEL_VERSION: 2.1.0
+    container: gcr.io/iree-oss/bazel-tensorflow
     steps:
       - name: Printing environment
         run: |
           "$CXX" --version
           "$CC" --version
           "$PYTHON_BIN" --version
-      - name: Installing bazel
-        run: |
-          # https://docs.bazel.build/versions/master/install-ubuntu.html
-          sudo apt-get install unzip zip
-          wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION?}/bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh
-          chmod +x bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh
-          ./bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh --user
-          rm bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh
-          echo "::add-path::${HOME?}/bin"
-      - name: Installing dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install clang libsdl2-dev python3 python3-pip python3-setuptools
-          # Update pip before using it so it picks up the latest versions of our dependencies.
-          sudo python3 -m pip install --upgrade pip
-          sudo python3 -m pip install numpy tf-nightly
-          # Install python2 numpy. Temporary fix for issue #1737.
-          sudo apt-get install -y python-pip
-          python -m pip install --upgrade pip
-          python -m pip install numpy
       - name: Checking out repository
         uses: actions/checkout@v2
-      - name: Initializing submodules
-        run: ./scripts/git/submodule_versions.py init
+        with:
+          submodules: true
```

Comment:
Our special way of installing submodules is load bearing. It initializes the submodules based on the version in SUBMODULE_VERSIONS, which we use as the source of truth for submodules because it's something that Copybara can actually track. At the time this was added, the v2 checkout action didn't have support for submodules (looks like it was added in https://github.com/actions/checkout/commit/422dc4567157f4d62b665a8a288310365b1d194b#diff-04c6e90faac2675aa89e2176d2eec7d8), which is why there's a special init command. Now, we should be able to use the init from the checkout action and then separately fix with any differences from the SUBMODULE_VERSIONS file. Probably better to do it that way if the checkout action has optimized submodule init. But that's a separate change, so I recommend dropping this part for now.

Reply:
SGTM